src/omneo/resources/profiles/index.ts

```diff
@@ -533,4 +534,17 @@ export default class Profiles extends Resource {
       return response.data
     })
   }
+
+  merge (sourceProfileID: string, destinationProfileId: string): Promise<Merge> {
```

Comment:
I believe the Omneo API returns the destination `Profile` as a response to `/merge`
Could you please confirm this?